src/common/transformations/src/transformations/transpose_sinking/ts_unsqueeze.cpp

```diff
@@ -190,9 +190,19 @@ TSUnsqueezeBackward::TSUnsqueezeBackward() {
                 return false;
             }
         } else {
-            auto rank = main_node->get_output_partial_shape(0).rank();
-            non_negative_axes =
-                util::try_get_normalized_axis_vector(unsqueeze_axes->get_tensor_view(), rank, *main_node);
+            const auto& axes = unsqueeze_axes->cast_vector<int64_t>();
+            if (std::all_of(axes.begin(), axes.end(), [](int64_t axis) {
```

Comment:
can it be ok if axes is empty? can we have test for it?

Reply:
according to specification, this case is possible. do we have such case? It can lead to dynamic rank, so may be it better to skip this transformation.

Reply:
We have to provide at least 1 axis if we create Unsqueeze with a Constant 
The shape inference will fail if it's empty
https://github.com/openvinotoolkit/openvino/blob/280d484aed58ae1855cde81d3cb4ded64c7737e0/src/core/shape_inference/include/unsqueeze_shape_inference.hpp#L35

I will add an additional check just in case.

Reply:
Sorry, I mixed with Squeeze operation. I think there is no need in additional check since Unsqueeze constructor should have already done this check before.
Thanks